src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.ts

```diff
@@ -93,9 +93,9 @@ export class TableComponent implements AfterContentChecked, OnInit, OnChanges, O
   // e.g. 'single' or 'multi'.
   @Input()
   selectionType: string = undefined;
-  // If `true` selected item details will be updated on table refresh
+  // By default selected item details will be updated on table refresh, if data has changed
   @Input()
-  updateSelectionOnRefresh = true;
+  updateSelectionOnRefresh: 'always' | 'never' | 'ifSelectedDataChanged' = 'ifSelectedDataChanged';
```

Comment:
What about using ``onChange`` instead of ``ifSelectedDataChanged``?

Reply:
Agree, I've changed it to `onChange`